### ISSUE TITLE
No visual focus indicator for back button

### ISSUE THREAD

**Issue by tyricec** [tyricec/qtp#2]
When tabbing through application and back button is enabled, there is no indicator that it is focused. The travel mode has one before it so it looks as if the focus disappears.

1. Create a directions result
2. Tab to Travel Mode dropdown
3. Press Tab

Expected Result

Back Button should have a focus outline

Actual Result

Back Button has no focus outline
